exasol_script_languages_container_tool/lib/api/generate_language_activation.py

```diff
@@ -0,0 +1,39 @@
+import textwrap
+from pathlib import Path
+
+from exasol_script_languages_container_tool.lib.tasks.upload.language_definition import LanguageDefinition
+
+
+def generate_language_activation(
```

Comment:
In general, for this we probably would need an api function, but it doesn't hurt either. The API here is the LanguageDefinition